client/app/services/auth.js

```diff
@@ -1,32 +1,40 @@
 import debug from 'debug';
+import { includes, extend } from 'lodash';
 
-const logger = debug('redash:auth');
```

Comment:
It's a bit hard to follow what changed here from the diff, can you summarize the changes?